.github/workflows/tests.yml

```diff
@@ -15,11 +15,16 @@ jobs:
     env:
       DB_USER: root
       DB_PASSWORD: root
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       fail-fast: false
       matrix:
         cfengine: [ "lucee@5", "adobe@2018", "adobe@2021" ]
         fullNull: [ "true", "false" ]
+        experimental: [false]
```

Comment:
Can you describe this magic :)

Reply:
Basically, we set `experimental` to `false` by default, and `true` for the Lucee 5.3 Release Candidate. Then, if `experimental` is `true`, we set `continue-on-error` to `true` so a buggy Lucee RC won't fail the build.

Reply:
We're doing the same to run CBORM against the Hibernate 5.4 extension without preventing CBORM releases if the Hibernate extension is buggy.

https://github.com/coldbox-modules/cborm/blob/development/.github/workflows/tests.yml#L24

Reply:
I had never seen that syntax before in github actions, do you happen to have a link to the docs so I can learn it?

Reply:
Sure. It's in the matrix documentation, under "Handling Failures".

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#handling-failures